sites/friday/src/migration/from_platformsh.md

```diff
@@ -0,0 +1,271 @@
+
+✓ Detected stack: Next.js
+✓ Detected runtime: JavaScript/Node.js
+✓ Detected dependency managers: Yarn
+Tell us your project name: [nextjs]
+
+                       (\_/)
+We're almost done...  =(^.^)=
+
+Last but not least, unless you're creating a static website, your project uses services. Let's define them:
+
+Which services are you using?
+Use arrows to move, space to select, type to filter
+  [x]  MariaDB
+  [ ]  MySQL
+  [ ]  PostgreSQL
+> [x]  Redis
+  [ ]  Redis Persistent
+  [ ]  Memcached
+  [ ]  OpenSearch
+
+You have not selected any service, would you like to proceed anyway? [Yes]
+
+┌───────────────────────────────────────────────────┐
+│   CONGRATULATIONS!                                │
+│                                                   │
+│   We have created the following files for your:   │
+│     - .environment                                │
+│     - .upsun/config.yaml                          │
+│                                                   │
+│   We're jumping for joy! ⍢                        │
+└───────────────────────────────────────────────────┘
+         │ /
+         │/
+         │
+  (\ /)
+  ( . .)
+  o (_(")(")
+
+You can now deploy your application to Upsun!
+To do so, commit your files and deploy your application using the
+Upsun CLI:
+  $ git add .
+  $ git commit -m 'Add Upsun configuration files'
+  $ upsun project:set-remote
+  $ upsun push
+```
+{{< note >}}
+as this command generate the minimum configuration to install the detected stack, you would probably need to compare your previous Yaml section from your `.platform.app.yaml` (or `.platform/applications.yaml`) file to your new `.upsun/config.yaml` file and report possible missing settings, like your additional command lines in your `build`/`deploy` hooks section.
+</br>When it’s done, commit and push your changes.
+{{< /note >}}
+
+## Defined app and services resources
+Within Platform.sh Yaml configuration files, it is possible to define resources of your app and services by using such settings.
+Example for your applications:
+```yaml
+# .platform.app.yaml
+name: app
+type: nodejs:16
+# How many resources to devote to the app. Defaults to AUTO in production environments.
+# More information: https://docs.platform.sh/create-apps/app-reference.html#sizes
+size: L
+# Fine-tune allocated resources
+# https://docs.platform.sh/create-apps/flexible-resources.html#performance-profiles
+resources:
+  base_memory: 1024
+  memory_ratio: 1024
+# The size of the persistent disk of the application (in MB). Minimum value is 128
+disk: 1024
+```
+
+Example for your services:
+```yaml
+# .platform/services.yaml
+database:
+  type: postgresql:15
+  disk: 1024
+```
+
+{{< note >}}
+These settings, size, disk and resources are no longer available in the Upsun Yaml config files (or not taken in account anymore).
+{{< /note >}}
+
+### Using the API
+#### Change app resources
+To change your `app` application resources, use the following command line:
```

Comment:
Per OVV, in a first build instance, if more than one service needs to be configured, you have to do both at once. You can't do the app, and then the database. We'll need to either update this section or, at a minimum, a warning/note.